src/PKSim.Core/Commands/AddMoleculeExpressionsFromQueryToSimulationSubjectCommand.cs

```diff
@@ -29,13 +29,14 @@ public override void Execute(IExecutionContext context)
          //First add the protein expression to the individual so that undo will be available
          Add(AddMoleculeToSimulationSubjectCommand(_molecule, _simulationSubject, context));
 
+         var allExpressions = _simulationSubject.AllExpressionParametersFor(_molecule);
```

Comment:
@Yuri05 In general, we now need the simulation subkect (individual or population) to perform any type of querying on parameters. We do not have a flat list anymore